src/library/styles/components/_link.scss

```diff
@@ -1,3 +1,43 @@
+// Default Link Styles
+a {
+  @include link-default;
```

Comment:
we can remove this from lib and move to docs (just the a @include link-default part, the classes can stay in lib)